File: sandbox/node/src/ReportCreateExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.ReportApi();
apiCaller.username = "YOUR_API_KEY";

const reportCreateRequest: models.ReportCreateRequest = {
  startDate: "09/01/2020",
  endDate: "09/01/2020",
  reportType: [
    models.ReportCreateRequest.ReportTypeEnum.UserActivity,
    models.ReportCreateRequest.ReportTypeEnum.DocumentStatus,
  ],
};

apiCaller.reportCreate(
  reportCreateRequest,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling ReportApi#reportCreate:");
  console.log(error.body);
});

File: sandbox/python/src/ReportCreateExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
)

with ApiClient(configuration) as api_client:
    report_create_request = models.ReportCreateRequest(
        start_date="09/01/2020",
        end_date="09/01/2020",
        report_type=[
            "user_activity",
            "document_status",
        ],
    )

    try:
        response = api.ReportApi(api_client).report_create(
            report_create_request=report_create_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling ReportApi#report_create: %s\n" % e)
